fs/fs.go

```diff
@@ -75,35 +75,9 @@ import (
 	"oras.land/oras-go/v2/content/oci"
 )
 
-const (
-	defaultFuseTimeout = time.Second
-	fusermountBin      = "fusermount"
-
-	// Amount of time the background fetcher will wait once a new layer comes in
-	// before (re)starting fetches.
-	defaultBgSilencePeriod = 30 * time.Second
-
-	// Specifies how often the fetch will occur.
-	// The background fetcher will fetch a single span every `defaultFetchPeriod`.
-	defaultBgFetchPeriod = 500 * time.Millisecond
-
-	// Specifies the maximum size of the bg-fetcher work queue i.e., the maximum number
-	// of span managers that can be queued. In case of overflow, the `Add` call
-	// will block until a span manager is removed from the workqueue.
-	defaultBgMaxQueueSize = 100
-
-	// The default amount of interval at which the background fetcher emits metrics
-	defaultBgMetricEmitPeriod = 10 * time.Second
-
-	// Amount of time Mount will time out if a layer can't be resolved.
-	defaultMountTimeout = 30 * time.Second
-
-	// Amount of time the snapshotter will wait before emitting the metrics for FUSE operation.
-	defaultFuseMetricsEmitWaitDuration = 60 * time.Second
-)
-
 var (
```

Comment:
nit: can these be `const`?

Reply:
No, since `SelectFirstPolicy` is a func.